src/views/Upload.vue

```diff
@@ -7,7 +7,7 @@
     </ion-header>
 
     <ion-content>
-      <ion-card v-for="product in uploadProducts" :key="product.productId">
+      <ion-card v-for="product in uploadProducts" :key="product.productId" @click="viewProduct(product)">
```

Comment:
Remove the click event from ion-label, as we have added the event on the card itself.